### PR TITLE
Update entrypoint and kindnetd

### PR DESCRIPTION
**Pull request by spowelljr**

### CODE REVIEW

**Comment by spowelljr** [comment]
ok-to-build-image

**Review by medyagh — APPROVED** [review]

File: deploy/kicbase/entrypoint [hunk 1]
```diff
@@ -102,22 +102,36 @@ overlayfs_preferrable() {
...
+      # we do not use the ZFS snapshotter because of skew issues vs the host
+      if [[ -e /dev/fuse ]]; then
+        snapshotter="fuse-overlayfs"
+      else
+        snapshotter="native"
+      fi
+    # fuse likely implies fuse-overlayfs, we should switch to fuse-overlayfs (or native)
+    elif [[ "$container_filesystem" == 'fuseblk' ]]; then
+      snapshotter="fuse-overlayfs"
     fi
```
Comment (criztovyl) [inline deploy/kicbase/entrypoint:131]: #15191

File: deploy/kicbase/entrypoint [hunk 2]
```diff
@@ -102,22 +102,36 @@ overlayfs_preferrable() {
...
+      # if fuse is present, use fuse-overlayfs, else fallback to native
+      # we do not use the ZFS snapshotter because of skew issues vs the host
+      if [[ -e /dev/fuse ]]; then
+        snapshotter="fuse-overlayfs"
+      else
+        snapshotter="native"
+      fi
+    # fuse likely implies fuse-overlayfs, we should switch to fuse-overlayfs (or native)
+    elif [[ "$container_filesystem" == 'fuseblk' ]]; then
+      snapshotter="fuse-overlayfs"
```
Comment (criztovyl) [inline deploy/kicbase/entrypoint:130]: #15191

**Pull request merged**